recipe/build.sh

```diff
@@ -1,8 +1,16 @@
 #!/bin/bash
 
+source activate "${CONDA_DEFAULT_ENV}"
+
 chmod +x configure
 
-./configure --prefix=$PREFIX --enable-cxx
+if [ "$(uname)" == "Darwin" ];
+then
+    ./configure --prefix=$PREFIX --enable-cxx --build=x86_64-apple-darwin --with-yasm=$PREFIX
+else
+    ./configure --prefix=$PREFIX --enable-cxx --enable-fat --with-yasm=$PREFIX
```

Comment:
Looks like they want the full path to the binary so `"${PREFIX}/bin/yasm"` presumably.